apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-close-click.js

```diff
@@ -9,7 +9,7 @@ import tracksRecordEvent from './track-record-event';
  * @returns {{handler: Function, selector: string, type: string}} event object definition.
  */
 export default () => ( {
-	selector: '.edit-post-fullscreen-mode-close__toolbar',
+	selector: '.edit-post-header .edit-post-fullscreen-mode-close',
```

Comment:
I think it's fine to keep this with the new selector and no longer track the old one.